Makefile

```
@@ -1359,6 +1359,10 @@ $(GOLANGCI_LINT): # Build golangci-lint from tools folder.
 $(GOVULNCHECK): # Build govulncheck.
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) $(GOVULNCHECK_PKG) $(GOVULNCHECK_BIN) $(GOVULNCHECK_VER)
 
+.PHONY: provider-issues
+provider-issues: # Creates issues for all CAPI providers 
```

probably this is clearer?:

```suggestion
provider-issues: # Creates GitHub issues in a pre-defined list of CAPI provider repositories
```

---

addressed via 756bcb44a70938a7b18f14951bf99c1658880640